index.js

```diff
@@ -85,11 +85,12 @@ export function mergeTrees(matchingTrees, context) {
 
 class Rule {
 
-    constructor(name, parent, style, filter) {
+    constructor(name, parent, style, filter, properties) {
```

Comment:
yeah, basically, you already have white-listed properties being passed through, but the constructors of the subclasses need to have each property name included explicitly. We could automate this by saving references to all white-listed property keys on the Rule objects (instead of having to explicitly assign each, like here: https://github.com/tangrams/unruly/pull/1/files#diff-168726dbe96b3ce427e7fedce31bb0bcR93). But really I was just noticing this in passing, in practice I'm not sure it's worth changing (can't imagine we're going to be adding lots of these things).